lib/dpd_api/requests/store_orders.rb

```diff
@@ -58,6 +58,10 @@ def xml_attributes
         attrs[:order_type] ||= "consignment"
         attrs[:saturday_delivery] ||= 0
 
+        attrs[:add_service] ||= 3 # 3 = written permission to deposit goods by Sender
+        attrs[:message_number] ||= 14
```

Comment:
what's this mean?

Reply:
I don't know, it was provided by DPD. Will try to find out